solutions/usaco-988.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-988
+source: USACO Bronze 2020 January
+title: Photoshoot
+author: Ananth Kashyap
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+data = open('photo.in').read().strip().split('\n')
+n = int(data[0])
+b = list(map(int, data[1].split(' ')))
```

Comment:
more descriptive variable names :)

Reply:
I'm pretty sure these var names are from the problem, so it should be relatively straightforwards as to what the prob is saying... 

Read this [comment](https://github.com/cpinitiative/usaco-guide/pull/1973#discussion_r675775588) for a more detailed reason.

Reply:
k then